.gitignore

```diff
@@ -24,5 +24,6 @@ Dockerfile.cross
 *.swp
 *.swo
 *~
+
+# secrets
```

Comment:
Strange. This change is in `main` already

Reply:
I created my branch on monday, and merged main changes yesterday. Maybe it is because of this.